packages/xsnap-lockdown/scripts/build-bundle.js

```diff
@@ -1,5 +1,7 @@
 #! /usr/bin/env node
-import '@endo/init';
+// Adopting legacy until Babel heals from property override mistakes.
+// https://github.com/endojs/endo/issues/1846
```

Comment:
```suggestion
// TODO Adopting legacy until Babel heals from property override mistakes.
// https://github.com/endojs/endo/issues/1846
```

See earlier question